### ISSUE TITLE
Hosts/Net always UNKNOWN

### ISSUE THREAD

**Issue by Ubiquitous** [gynter/yPcalc#1]
On `OS X` running `Python 2.7.6`:

```
$ ypcalc show 74.125.239.135/32
Address:     74.125.239.135/32
Netmask:     255.255.255.255 = 32
Network:     74.125.239.135/32
HostMin:     74.125.239.135/32
HostMax:     74.125.239.135/32
Broadcast:   74.125.239.135
Hosts/Net:   1 UNKNOWN
IPv6 repr:   2002:4a7d:ef87:0000:0000:0000:0000:0000
PTR RR name: 135.239.125.74.in-addr.arpa
IP version:  4
```

It seems any outside ip addresses `Hosts/Net` is labeled as `UNKNOWN` unless it's a private one such as `192.168.0.1`:

```
$ ypcalc show 192.168.0.1
Address:     192.168.0.1/32
Netmask:     255.255.255.255 = 32
Network:     192.168.0.1/32
HostMin:     192.168.0.1/32
HostMax:     192.168.0.1/32
Broadcast:   192.168.0.1
Hosts/Net:   1 PRIVATE
IPv6 repr:   2002:c0a8:0001:0000:0000:0000:0000:0000
PTR RR name: 1.0.168.192.in-addr.arpa
IP version:  4
```

Is this normal behavior? I noticed in the `readme` there was different information shown such as `Hosts/Net:   32 CLASS C`, or `633825300114114700748351602688 GEO-UNICAST`, although I only seem to get the output of `UNKNOWN` or `PRIVATE`.

**Comment by gynter** [comment]
I think that this behavior is intented, see http://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml for the descriptions.

It used to work, since the data in the tehmaze/ipcalc library was out of date, but was fixed with tehmaze/ipcalc@e654c6e7688090b80b33c0bab717a2ea8ad76814 (tehmaze/ipcalc#33)

I also updated the examples in README.

**Comment by Ubiquitous** [comment]
I appreciate the quick response; it makes perfect sense. 

**yPcalc** is very useful — thanks very much :)
